tests/bluetooth/shell/src/main.c

```diff
@@ -26,17 +26,13 @@
 
 #include <gatt/hrs.h>
 
-#define print(_sh, _ft, ...) \
+#define print_sh(_sh, _ft, ...) \
```

Comment:
WHy is this rename needed as part of this PR?

Reply:
I was affraid that `print` and `printk` are too similar.

Reply:
I don't think that's an issue. Can you remove that change?

Reply:
Well yes, because it's unrelated to this particular problem you are solving with the PR here